R/get_predicted_se.R

```diff
@@ -13,7 +13,7 @@ get_predicted_se <- function(x,
   # the diagonal of this matrix represent the standard errors of the predictions,
   # which are then multiplied by 1.96 for the confidence intervals.
 
-  vcovmat <- .get_predicted_ci_vcov(
+  vcovmat <- .get_varcov_sandwich(
```

Comment:
Shouldn't this call `get_varcov()`, and then automatically use sandwich estimators if required, and normal vcov if not?

Reply:
ebe8d2e4